src/store/itemShare.ts

```diff
@@ -32,24 +33,22 @@ export const useShareStore = defineStore("share", {
       })
     },
 
-    // TODO:
-    // async deleteItem(item: Item, podcastname: string, from: string) {
-    // })
-
-    async sendItem(item: Item, destination: string, from: string) {
-      // const db = getFirestore()
-      const docRef = doc(db, destination, "inbox", from, "shares")
-      try {
-        await updateDoc(docRef, {
-          items: arrayUnion(item.text),
-        })
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      } catch (e: any) {
-        // create doc if it doesn't exist
-        if (e.code === "not-found" && e.name === "FirebaseError")
-          await setDoc(docRef, { items: arrayUnion(item.text) })
-        else throw e
-      }
+    async sendItem(item: Item, destination: [], from: string) {
+      destination.forEach(async (podcast) => {
+        const docRef = doc(db, podcast, "inbox", from, "shares")
+        try {
+          await updateDoc(docRef, {
+            items: arrayUnion(item.text),
+          })
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        } catch (e: any) {
+          // create doc if it doesn't exist
+          if (e.code === "not-found" && e.name === "FirebaseError")
+            await setDoc(docRef, { items: arrayUnion(item.text) })
+          else throw e
+        }
+      })
+      this.connect(from)
```

Comment:
?

Reply:
After switching podcasts in the podcast menu then all connections are mixed so after sharing we need to call the current connection So I have called the "connect" function hear we have removed this function and then got an inbox reordering issue please refer to this issue in my attachment.

https://user-images.githubusercontent.com/13918292/170646958-fcfedac9-e17e-4898-9d28-ec0214fc55cd.mp4

Reply:
I see, sounds like a work around to a bug, this shouldn't really be needed